optimum/utils/preprocessing/token_classification.py

```diff
@@ -28,7 +28,7 @@
 
 class TokenClassificationProcessing(TaskProcessor):
     ACCEPTED_PREPROCESSOR_CLASSES = (PreTrainedTokenizerBase,)
-    DEFAULT_DATASET_ARGS = "conll2003"
+    DEFAULT_DATASET_ARGS = {"path": "conll2003", "trust_remote_code": True}
```

Comment:
we shouldn't set `trust_remote_code` to True by default here, where is this needed ?

Reply:
this dataset was being used with "remote code" (it generates the dataset) for as long as it existed, but the `datasets` package introduced the new argument `trust_remote_code` lately, this preprocessing is used in in tflite tests, we can either find a similar dataset that doesn't use code or allow remote code (that was already being executed).
https://huggingface.co/datasets/eriktks/conll2003/blob/main/conll2003.py

Reply:
I think we need to change the dataset to something that doesn't require `trust_remote_code` to be set to True by default then (something that should be explicitly set by users)